src/borg/testsuite/logger.py

```diff
@@ -39,7 +163,7 @@ def test_parent_module():
     assert find_parent_module() == __name__
 
 
-def test_lazy_logger():
+def test_lazy_logger(io_logger):
```

Comment:
hm?

Reply:
All of borgs testing code relies on a global logging setup defined in conftest.py. But when testing the logging setup itself, it makes sense to reset it after every test function. My new teardown_function() does that.
Thus ``test_lazy_logger()`` doesn't have a logging setup anymore.
An alternative would be to call ``setup_logging()`` at the beginning.